readme.md

```diff
@@ -26,6 +32,14 @@ To install Administrator as a Composer package to be used with Laravel 4, simply
 )
 ```
 
+### Laravel 5
+
+Then publish the config file with `php artisan vendor:publish`. This will add the file `app/config/administrator.php`. This [config file](http://administrator.frozennode.com/docs/configuration) is the primary way you interact with Administrator.
+
+Next to that all the assets, views and translation files will be published too.
+
+### Laravel 4
+
```

Comment:
I think only the Form facade is being used in:

- [src/views/settings.php](https://github.com/FrozenNode/Laravel-Administrator/blob/da269970a0b1153191cbaa76dbadd0bda6757eb1/src/views/settings.php#L25)
- [src/views/index.php](https://github.com/FrozenNode/Laravel-Administrator/blob/346105fa651ea8b1a30c5393643a13289f5f84c8/src/views/index.php#L70)